### PR TITLE
Adds Initial Tests by re-using storybooks as tests

### PR DESCRIPTION
**Pull request by orta**
Also ensures a consitent Flow coverage, allows you to do #5 - will comment inline

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -23,16 +23,7 @@
...
-    "flowtype/define-flow-type": 1,
-    "flowtype/require-parameter-type": [1, { "excludeArrowFunctions": true }],
-    "flowtype/require-return-type": [ 0,"always", { "annotateUndefined": "never" } ],
-    "flowtype/space-after-type-colon": [ 1, "always" ],
-    "flowtype/space-before-type-colon": [ 1, "never" ],
-    "flowtype/type-id-match": [ 0, "^([A-Z][a-z0-9]+)+Type$" ],
-    "flowtype/use-flow-type": 1,
-    "flowtype/valid-syntax": 1,
-    "react/jsx-uses-react": "error"
+    "comma-dangle": ["error", "always-multiline"]
```
Comment (orta) [inline .eslintrc.json:26]: the "extends" plugin thing is new, which is why we don't use in emission - ^ these are all basically the recommended anyway

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      "runtimeExecutable": null,
+      "runtimeArgs": [
+        "--debug-brk",
+        "./node_modules/.bin/jest",
+        "-i"
+      ],
+      "cwd": "${workspaceRoot}"
+    }
+  ]
+}
```
Comment (orta) [inline .vscode/launch.json:37]: Not done anything WRT the Launc / Attach, but the test one I used

File: app/_tests/__snapshots__/storyshots.test.js.snap [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  <div>
+    <i>
+      OK, b. 1999
+    </i>
+  </div>
+  <i>
+    12 Followers
+  </i>
+</div>
+`;
```
Comment (orta) [inline app/_tests/__snapshots__/storyshots.test.js.snap:15]: This is what the stubbed data one looks like

File: app/_tests/storyshots.test.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import initStoryshots from 'storyshots'
+
+initStoryshots({
+  storyRegex: /^((?!(r|R)elay).)*$/,
+})
```
Comment (orta) [inline app/_tests/storyshots.test.js:5]: only run tests that don't have either "Relay" or "relay" in the name

File: package.json [hunk 5]
```diff
@@ -8,12 +8,14 @@
...
-    "eslint": "eslint",
+    "test": "eslint . && flow . && jest",
     "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/master && yarn install && npm run dump-schema -- ../../data",
     "create-types": "gql2ts data/schema.json",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "flow": "flow",
+    "eslint": "eslint",
+    "jest": "jest"
```
Comment (orta) [inline package.json:18]: https://twitter.com/orta/status/814174686636625920

File: app/routes.js [hunk 6]
```diff
@@ -14,7 +17,7 @@ const app = express.Router()
 
 app.use(artsyRelayMiddleware)
 
-app.get('/artist/:id', (req: $Request, res: $Response, NextFunction) => {
+app.get('/artist/:id', (req: $Request, res: $Response, next: NextFunction) => {
```
Comment (alloy) [inline app/routes.js:20]: Do you actually get useful type info from these interfaces? I just got `any`

**Comment by alloy** [comment]
Nice

File: app/routes.js (hunk 6 above)
Comment (orta) [reply app/routes.js:20]: Yep

screen shot 2016-12-29 at 10 17 42 am

Comment (alloy) [reply app/routes.js:20]: Weird, I’ll try again.

Comment (orta) [reply app/routes.js:20]: if it doesn't we might want to set this on the project settings in VS Code

screen shot 2016-12-29 at 2 17 30 pm

**Pull request merged**